qutebrowser/browser/browsertab.py

```diff
@@ -618,6 +622,42 @@ def load_items(self, items: typing.Sequence) -> None:
         raise NotImplementedError
 
 
+class AbstractHistoryItem:
+
+    """A single item in the tab history.
+
+    Attributes:
+        url: The QUrl of this item.
+        original_url: The QUrl of this item which was originally requested.
+        title: The title as string of this item.
+        active: Whether this item is the item currently navigated to.
+        user_data: The user data for this item.
+    """
+
+    def __init__(self, url: QUrl, title: str, *, original_url: QUrl = None,
```

Comment:
My preference would be to split each param into it's own line... but i followed convention here.

Reply:
I also prefer one item per line. I don't think there's a strict convention – even this file has many styles.